sdk/src/main/java/dev/cdevents/constants/CDEventConstants.java

```diff
@@ -15,14 +15,24 @@ private CDEventConstants() {
     public static final String SPEC_REPO = ".." + File.separator + "spec";
 
     /**
-     * Event JsonSchema files location.
+     * Event JsonSchema file system location.
      */
     public static final String SCHEMA_FOLDER = SPEC_REPO + File.separator + "schemas";
 
+    /**
+     * Event JsonSchema classpath location.
+     */
+    public static final String SCHEMA_CLASSPATH = "classpath:dev/cdevents/spec/schemas/";
+
+    /**
+     * Event link schemas location.
+     */
+    public static final String SCHEMA_LINKS_FOLDER = SCHEMA_FOLDER + File.separator + "links";
+
     /**
      * CDEvents Version.
      */
-    public static final String CDEVENTS_SPEC_VERSION = "0.3.0";
+    public static final String CDEVENTS_SPEC_VERSION = "0.4.0";
```

Comment:
is it `0.4.0` or `0.4.1`?

Reply:
yes it is 0.4.1, I missed it, thanks for catching this.